tests/helpers.py

```diff
@@ -16,9 +18,23 @@ def expected_data_within_get_response(
     """
     response = test_client.get(endpoint, follow_redirects=True)
     response_data = json.loads(response.data)
-    print(endpoint)
-    print(response_data)
-    assert response_data == expected_data, response_data
+
+    if isinstance(expected_data, List):
```

Comment:
Ok, I've got to be honest - this is quite hard to read and understand. Any chance of adding some comments explaining what is happening? Or perhaps I could challenge you to write this in a more pythonic and elegant way?

Reply:
Yep is horrific! I need a nested solution to go into two dictionaries and iterate through items and check matches for both dicts and lists - will work on

Reply:
Not exactly rising to the challenge - but found a package DeepDiff that does deep dictionary comparison, which should be more manageable